Marlin/src/config/examples/Anet/A6/Configuration.h

```diff
@@ -571,9 +571,32 @@
 #define Z_MAX_ENDSTOP_INVERTING false // set to true to invert the logic of the endstop.
 #define Z_MIN_PROBE_ENDSTOP_INVERTING false // set to true to invert the logic of the probe.
 
+/**
+ * Specify Stepper Driver types
+ * The options are used to determine driver pulse timings as well as more advanced functionality.
+ * The timings can be overriden in Configuration_adv.h
+ *
+ * Options: A4988, DRV8825, LV8729, TB6560, TB6600, TMC2100,
+ *          TMC2130, TMC2130_STANDALONE, TMC2208, TMC2208_STANDALONE,
+ *          TMC26X,  TMC26X_STANDALONE,  TMC2660, TMC2660_STANDALONE,
+ *          TMC5130, TMC5130_STANDALONE
+ * :['A4988','DRV8825','LV8729','TB6560','TB6600','TMC2100','TMC2130','TMC2130_STANDALONE','TMC2208','TMC2208_STANDALONE','TMC26X','TMC26X_STANDALONE','TMC2660','TMC2660_STANDALONE','TMC5130','TMC5130_STANDALONE']
+ */
+#define X_DRIVER_TYPE  A4988
+#define Y_DRIVER_TYPE  A4988
+#define Z_DRIVER_TYPE  A4988
+//#define X2_DRIVER_TYPE A4988
+//#define Y2_DRIVER_TYPE A4988
+//#define Z2_DRIVER_TYPE A4988
+#define E0_DRIVER_TYPE A4988
+//#define E1_DRIVER_TYPE A4988
+//#define E2_DRIVER_TYPE A4988
+//#define E3_DRIVER_TYPE A4988
+//#define E4_DRIVER_TYPE A4988
+
 // Enable this feature if all enabled endstop pins are interrupt-capable.
 // This will remove the need to poll the interrupt pins, saving many CPU cycles.
-#define ENDSTOP_INTERRUPTS_FEATURE
+//#define ENDSTOP_INTERRUPTS_FEATURE
```

Comment:
I see in a few configurations that `ENDSTOP_INTERRUPTS_FEATURE` is being disabled. Is this because the pins that go with these configs are not interrupt-capable?

Reply:
Not intentional. I'll need to redo the search&replace operation.